backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr-doc.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-creating-backup-cr-doc"]
+= Creating a Backup CR
+include::_attributes/common-attributes.adoc[]
+:context: backing-up-applications
+
+toc::[]
+
+You back up Kubernetes images, internal images, and persistent volumes (PVs) by creating a `Backup` custom resource (CR).
+
+.Prerequisites
+
+* You must install the OpenShift API for Data Protection (OADP) Operator.
+* The `DataProtectionApplication` CR must be in a `Ready` state.
+* Backup location prerequisites:
+** You must have S3 object storage configured for Velero.
+** You must have a backup location configured in the `DataProtectionApplication` CR.
+* Snapshot location prerequisites:
+** Your cloud provider must have a native snapshot API or support Container Storage Interface (CSI) snapshots.
+** For CSI snapshots, you must create a `VolumeSnapshotClass` CR to register the CSI driver.
+** You must have a volume location configured in the `DataProtectionApplication` CR.
+
+.Procedure
+
+. Retrieve the `backupStorageLocations` CRs by entering the following command:
+
++
+[source,terminal]
+----
+$ oc get backupStorageLocations -n openshift-adp
+----
++
+.Example output
++
+[source,terminal]
+----
+NAMESPACE       NAME              PHASE       LAST VALIDATED   AGE   DEFAULT
+openshift-adp   velero-sample-1   Available   11s              31m
+----
+
+. Create a `Backup` CR, as in the following example:
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup>
+  labels:
+    velero.io/storage-location: default
+  namespace: openshift-adp
+spec:
+  hooks: {}
```

Comment:
there is a separate section for hooks, correct?

Reply:
Yes. But I need to do the links after I merge everything